### ISSUE TITLE
mongod.conf file missing

### ISSUE THREAD

**Issue by saigop** [RocketChat/Rocket.Chat#5079]
Your Rocket.Chat version: 0.46.0
Ubuntu Version - 16.04
Installed via snap package manager

Hope mongodb is the tool used for storage. Am trying to monitor the mongodb service remotely, but trying to see the open port 27017, the connection refused, while checking netstat the port is not open for 0.0.0.0

To enable this, I need to open in mongod.conf file, which I See the file is missing, I tried this in 2 installation with same Ubuntu 16.04 and installed via snap. 

But still I see the conf file is missing. Do i need to create manually?

Thanks in advance.

**Comment by lucasvanhalst** [comment]
Opening up your database to the internet is one of the worst things you can do, just don't do it. ssh into your machine and connect to it that way if you so desire.

**Comment by saigop** [comment]
Agree, but basically I want to monitor the rocket chat service and mongodb service from icing a for any production impact.

**Comment by saigop** [comment]
Or even for monitoring I can bindip only the monitoring instance alone. Even for that too, I have to enter into mongod.conf file.

**Comment by geekgonecrazy** [comment]
@saigop yes we actually set it to only listen on the loopback interface.  Lets change the title of this and make it into a feature request, as to fix this for you we would have to allow the changing of listen interfaces.

**Comment by mehdiyahyaoui** [comment]
Hello

Any development on this open issues... ?

Thnx

**Comment by geekgonecrazy** [comment]
None currently.  We'll reference this issue when we open a PR that closes this.

**Comment by debdutdeb** [comment]
Available in the later releases.
